src/auth/jwtaccess.ts

```diff
@@ -71,10 +78,12 @@ export class JWTAccess {
       secret: this.key
     };
 
-    // Sign the jwt and invoke metadataCb with it.
+    // Sign the jwt and add it to the cache
     const signedJWT =
         jws.sign({header: {alg: 'RS256'}, payload, secret: this.key});
-    return {headers: {Authorization: 'Bearer ' + signedJWT}};
+    const res = {headers: {Authorization: 'Bearer ' + signedJWT}};
```

Comment:
nit: as long as you're changing this, switch to using a template string instead of using concatenation.

Reply:
fixed!